Source/ThirdParty/libwebrtc/Source/webrtc/sdk/objc/components/video_codec/RTCVideoDecoderH264.mm

```diff
@@ -161,7 +161,7 @@ - (NSInteger)decodeData:(const uint8_t *)data
   rtc::ScopedCFTypeRef<CMVideoFormatDescriptionRef> inputFormat =
       rtc::ScopedCF(webrtc::CreateVideoFormatDescription(data, size));
   if (inputFormat) {
-    _reorderQueue.setReorderSize(webrtc::ComputeH264ReorderSizeFromAnnexB(data, size));
+    _reorderQueue.setReorderSize(_useAVC ? webrtc::ComputeH264ReorderSizeFromAnnexB(data, size) : 0);
```

Comment:
Please add a FIXME comment to remove this workaround once MS Teams fixes their streaming app.

Reply:
Done